.tmp/tasks.md

```
@@ -492,16 +493,17 @@
+- [ ] サイドバーナビゲーション（Overview, Developers, Campaigns, Funnel）。一番下に System Settings へのアイコン
 - [ ] ヘッダー（ロゴ、ユーザー情報）
-- [ ] TailwindCSSでレスポンシブデザイン
+- [ ] サイドバーの項目、ウィジェットはプラグイン次第で増減することを想定
 - **完了条件**: ダッシュボードが表示され、ナビゲーションが機能する
 - **依存**: Task 2.1
 - **推定時間**: 3時間
 
 ### Task 7.2: Overviewページ実装
 
 - [ ] `app/routes/dashboard/overview.tsx`作成
+- [ ] Swapyでウィジェットをドラッグ＆ドロップ可能に（https://swapy.tahazsh.com/）
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)